src/Sentry/Sentry.csproj

```diff
@@ -49,6 +49,10 @@
     <PackageReference Include="System.Reflection.Metadata" Version="5.0.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\Sentry.Analyzers\Sentry.Analyzers.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```

Comment:
Does this need `PrivateAssets>all` ?

Reply:
If I understand the [docs](https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets) correctly, it could have even been fine, but I added it anyway. I also added private assets in analyzer references.